pkg/webhook/controlplane/ensurer.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/gardener/gardener/extensions/pkg/webhook/controlplane/genericmutator"
 	v1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
 	extensionsv1alpha1 "github.com/gardener/gardener/pkg/apis/extensions/v1alpha1"
+	kubernetesutils "github.com/gardener/gardener/pkg/utils/kubernetes"
```

Comment:
Nit for consistency:

```suggestion
	kutil "github.com/gardener/gardener/pkg/utils/kubernetes"
```